x-pack/plugin/sql/src/main/resources/org/elasticsearch/xpack/sql/plugin/sql_whitelist.txt

```diff
@@ -98,8 +98,10 @@ class org.elasticsearch.xpack.sql.expression.function.scalar.whitelist.InternalS
 #
```

Comment:
Seems we are missing the existing non-iso methods here, and relevant tests that would catch those cases.

Reply:
The iso methods are handled through the `dateTimeChrono` method which also handles the rest of the date time functions. I've renamed the `DateTimeExtractor` enum values for ISO day_of_week and week_of_year functions, hopefully making this less confusing.